src/System.Management/pkg/System.Management.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Management.csproj">
-      <SupportedFramework>netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp2.0;net45;$(UAPvNextTFM);$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Do we need this in other libraries? Only 5 libraries already have it.

And System.Drawing.Common.pkproj eg  doesn't have net45/net461 either:
`<SupportedFramework>netcoreapp2.0</SupportedFramework>`

@safern I wonder if all these need regularizing as well.

Reply:
> System.Drawing.Common.pkproj eg doesn't have net45/net461 either.

I'm adding net461 into System.Drawing.Common and also AllXamarinFrameworks
UAPvNextTFM I don't think we need it since it will already be netstandard. 

Yes, this need regularizing as well from the quick look I just took. I can work with @pjanotti offline to get it according to what we have in the other projects.

Reply:
Per my conversation with Tarek that is my understanding: this needs regularizing.

UAPvNextTFM was added at this moment while adding the PNS assembly for uap, otherwise we hit: "System.Management should not be supported on UAP,Version=v10.0.15138/win10-x86 but has runtime assets "

Alternatively we can keep it closer to `System.DirectoryServices.AccountManagement` that doesn't have a PNS assembly for uap yet.

That said I want to make this initial commit soon so I can focus more on the test coverage and the sources themselves, so far I've been most of the time dealing with build/packaging issues.

Reply:
Maybe the UapVNextTFM is then needed. However we need netfx facades also. You can merge as is and I can revisit and fix the configurations if you want. But fixing them wouldn’t take us more than 1 hour.

Reply:
You can take this PR as an example: https://github.com/dotnet/corefx/pull/24758